src/YesSql.Core/Services/DbBlockIdGenerator.cs

```diff
@@ -50,36 +51,40 @@ public async Task InitializeAsync(IStore store)
             UpdateCommand = "UPDATE " + _dialect.QuoteForTableName(_tablePrefix + TableName, _schema) + " SET " + _dialect.QuoteForColumnName("nextval") + "=@new WHERE " + _dialect.QuoteForColumnName("nextval") + " = @previous AND " + _dialect.QuoteForColumnName("dimension") + " = @dimension;";
             InsertCommand = "INSERT INTO " + _dialect.QuoteForTableName(_tablePrefix + TableName, _schema) + " (" + _dialect.QuoteForColumnName("dimension") + ", " + _dialect.QuoteForColumnName("nextval") + ") VALUES(@dimension, @nextval);";
 
-            await using (var connection = store.Configuration.ConnectionFactory.CreateConnection())
-            {
-                await connection.OpenAsync();
+            await using var connection = store.Configuration.ConnectionFactory.CreateConnection();
+            await connection.OpenAsync();
 
-                try
-                {
-                    await using (var transaction = connection.BeginTransaction(store.Configuration.IsolationLevel))
-                    {
-                        var localBuilder = new SchemaBuilder(store.Configuration, transaction, false);
+            try
+            {
+                await using var transaction = connection.BeginTransaction(store.Configuration.IsolationLevel);
+                var localBuilder = new SchemaBuilder(store.Configuration, transaction, false);
 
-                        localBuilder.CreateTable(TableName, table => table
-                            .Column<string>("dimension", column => column.PrimaryKey().NotNull())
-                            .Column<long>("nextval")
-                            );
+                await localBuilder.CreateTableAsync(TableName, table => table
+                    .Column<string>("dimension", column => column.PrimaryKey().NotNull())
+                    .Column<long>("nextval")
+                    );
 
-                        await transaction.CommitAsync();
-                    }
-                }
-                catch
-                {
-
-                }
+                await transaction.CommitAsync();
+            }
+            catch
+            {
+            }
+            finally
+            {
+                await connection.CloseAsync();
```

Comment:
Not necessary as `await using` will call `DisposeAsync`, which would then call `CloseAsync`.

Reply:
removed